modules/cluster-logging-elasticsearch-limits.adoc

```diff
@@ -37,8 +37,28 @@ spec:
             cpu: "4000m"
             memory: "4Gi"
           requests:
-            cpu: "100m"
+            cpu: "4000m"
             memory: "1Gi"
 ----
 <1> Specify the CPU and memory limits as needed. If you leave these values blank,
 the Elasticsearch Operator sets default values that should be sufficient for most deployments.
++
+If you need to adjust the amount of Elasticsearch CPU and memory, you must change both the request value and the limit value. 
++
+For example:
++
+[source,yaml]
+----
+      resources:
```

Comment:
Why is the syntax different here an in the previous YAML snippet? Can you omit the "" like this?